src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1387,9 +1387,26 @@ private IOperation CreateBoundLocalDeclarationOperation(BoundLocalDeclaration bo
             }
             else
             {
-                // we can get here if someone asked about 1 variable declarator on multi local declaration
-                Debug.Assert(boundLocalDeclaration.Syntax.Kind() == SyntaxKind.VariableDeclarator);
-                return CreateVariableDeclaration(boundLocalDeclaration, boundLocalDeclaration.Syntax);
+                var declarator = (VariableDeclaratorSyntax)boundLocalDeclaration.Syntax;
```

Comment:
> var declarator = (VariableDeclaratorSyntax)boundLocalDeclaration.Syntax; [](start = 15, length = 73)

It doesn't feel safe to make an assumption what syntax is associated with the bound node.

Reply:
okay, I will make it to handle case where syntax is not variable declarator syntax